ASSESSMENT.md

```diff
@@ -0,0 +1,100 @@
+# Assessment
+
+Candidates should assume that the solution will be deployed to an empty cloud subscription with no existing infrastructure in place.
+
+There *should not* be a requirement for Vibrato to access a candidate's cloud services account to deploy this solution.
+
+Demonstrate regular commits and good git workflow practices.
+
+There is no time limit for this test.
+
+Candidates should provide documentation on their solution, including:
+
+- Pre requisites for your deployment solution.
+- High level architectural overview of your deployment.
+- Process instructions for provisioning your solution.
+
+## Assessment Grading Criteria
+
+### Key Criteria
+
+Candidates should take care to ensure that thier submission meets the following criteria:
+
+- Must be able to start from a cloned git repo.
+- Must document any pre-requisites clearly.
+- Must be contained within a GitHub project.
+- Must deploy via an automated process.
+
+### Grading
+
+Candidates will be assessed across the following categories:
+
+#### Coding Style
+
+- Clarity of code
+- Comments where relevant
+- Consistency of Coding
+
+#### Security
+
+- Network segmentation
+- Secret storage
+- Platform security features
+
+#### Simplicity
+
+- No superfluous dependencies
+- Do not over engineer the solution
+
+#### Resiliency
+
+- Auto scaling and highly available frontend
+- Highly available Database
+
+## Tech Test Application
```

Comment:
Probably don't need this and everything below it here. I think it should be covered in the README, and is more application related rather than assessment related (i.e. assessment instructions and criteria)

Some of it, such as "Compiling from source" actually is already covered in the README